src/lib/application.ts

```diff
@@ -286,16 +286,28 @@ export class Application extends AbstractComponent<
         if (!this.internationalization.hasTranslations(this.lang)) {
             // Not internationalized as by definition we don't know what to include here.
             this.logger.warn(
-                `Options specified "${this.lang}" as the language to use, but TypeDoc does not support it.` as TranslatedString,
+                `Options specified "${this.lang}" as the language to use, but TypeDoc cannot provide translations for it.` as TranslatedString,
             );
-            this.logger.info(
-                ("The supported languages are:\n\t" +
-                    this.internationalization
-                        .getSupportedLanguages()
-                        .join("\n\t")) as TranslatedString,
-            );
-            this.logger.info(
-                "You can define/override local locales with the `locales` option, or contribute them to TypeDoc!" as TranslatedString,
+            if (this.lang !== "jp") {
```

Comment:
Adding a warning for use of `jp` seems reasonable to me, but this over complicates things. It should still be supported for now and listed in the supported languages list.

Reply:
This is not run _now_ because jp.cts _still_ exists.
It will be run once you remove jp.cts.

Reply:
Which I won't until the next minor release. I don't want to include dead code in planning for a future deprecation.

Reply:
I see. How about commenting it out instead of removing it?

Reply:
No, commented out code is still dead code that shouldn't exist, it can be brought back if necessary in the future. When jp is removed, I will likely choose not to keep this message. It will be documented as a breaking change in the changelog, and the warning will have been around for several months at that point.

Reply:
I understand your plan.